doc/nrf/templates/Integration_template_README.rst

```diff
@@ -1,41 +1,41 @@
-.. _Ecosystem_integration_template:
+.. _integration_template:
 
-Template: Ecosystem integration
-###############################
+Template: Integration
+#####################
 
 .. contents::
    :local:
    :depth: 2
 
 .. note::
-   Provide a short introductory description of the tool or service (third-party), indicating the value it brings to |NCS|.
+   Provide a short introductory description of the product (third-party), indicating the value it brings to |NCS|.
    Refer to the following instructions to create the description:
 
-   * Provide a title that starts with the tool or service name with "integration" as a suffix, for example, "Memfault integration".
+   * Provide a title that starts with the product name with "integration" as a suffix, for example, "Memfault integration".
    * Place the documentation inside the :file:`nrf/doc/nrf` folder.
-   * List the file name with path in the ``.. toctree:`` of the :file:`ecosystem_integration` RST file.
+   * List the file name with path in the ``.. toctree:`` of the :file:`integrations` RST file.
    * Sections with * are optional and can be left out.
      All other sections are required.
      There could be more sections, which can be added after discussion with the tech writer team.
      However, it should have the mandatory sections from the template in the recommended format at the very least.
 
 .. tip::
-   Describe the tool or service in one or two sentences maximum (full sentences, not sentence fragments), so that the user gets a clear idea of what the tool or service does.
-   This introduction must be concise and clear, highlighting the features of the tool or service that it brings to |NCS|.
+   Describe the product in one or two sentences maximum (full sentences, not sentence fragments), so that the user gets a clear idea of what the product does.
+   This introduction must be concise and clear, highlighting the features of the product that it brings to |NCS|.
 
 Partner overview*
 *****************
 
 .. note::
-   Provide an explanation of the tool or service.
-   This is optional if the tool or service is described above.
+   Provide an explanation of the product.
+   This is optional if the product is described above.
 
 Prerequisites
 *************
 
 .. note::
-   * Provide prerequisites that must be completed before the integration with the tool or service.
-   * Mention any other tool or service integration that needs to be completed.
+   * Provide prerequisites that must be completed before the integration with the product.
+   * Mention any other product integration that needs to be completed.
 
 Before you start the |NCS| integration with AVSystem's Coiote IoT Device Management, make sure that the following prerequisites are completed:
```

Comment:
If we're taking the AVSystem integration as the example here, I would suggest giving it another pass, as it is mixing product with provider in the first two sections (short desc and overview) in its current docs: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/external_comp/avsystem.html

"AVSystem is a software solution provider that provides a device management platform. "
[...]
"AVSystem provides the following features as a solution partner for Nordic Semiconductor’s cellular devices"

Shouldn't it be "The AVSystem’s Coiote IoT Device Management provides"? Current phrasing could suggest there are more products we're talking here...

Reply:
I think you are correct.

I would also say that if we have different products from the same vendor integrated with NCS then these should be described separately.

Reply:
I shall add the changes for the AVSystem user guide in a separate PR.